### DISCUSSION TITLE
Welcome to tg-spam Discussions!

### DISCUSSION THREAD

**Discussion by umputun** [umputun/tg-spam#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cherkoM** [comment]
Hello @umputun, 
Could you please clarify if TG-Spam supports telegram groups with topics (themes)?

**Reply by umputun** [reply]
hi. I have never tried it this way, but I believe it should be working. You can try to run it with the `--dbg` flag and see if it gets all the messages. As I said - I don't see a reason why it won't, as the API it uses doesn't do anything special to ignore such messages

**Reply by cherkoM** [reply]
@umputun thank you for the reply!
